### PR TITLE
perf(Boards, MiscDrivers):  Reduce refresh time and add font color selection on TFT Display drivers for MAX32690

### PR DESCRIPTION
**Pull request by crsz20**
### Description

Slight modifications to the TFT LCD display drivers on the MAX32690 EV Kit V1 board.
- Removed delays in `TFT_SPI_Write()` to reduce the screen refresh time from 0.87129 seconds to 0.120858 seconds -- an 86% improvement.
- `MXC_TFT_PrintFont()` would use the initially set background color around the printed text. Added a function to offer color selection of both the background and the font text.

### Checklist Before Requesting Review

- [x] PR Title follows correct guidelines.
- [x] Description of changes and all other relevant information.
- [ ] (Optional) Link any related GitHub issues [using a keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)
- [ ] (Optional) Provide info on any relevant functional testing/validation.  For API changes or significant features, this is not optional.

### CODE REVIEW

File: Libraries/MiscDrivers/Display/tft_st7735.h [hunk 1]
```diff
@@ -216,12 +216,25 @@ void MXC_TFT_ResetCursor(void);
  *
  * @param       x0              x location on screen
  * @param       y0              y location on screen
- * @param       fon_id          Font number
+ * @param       id              Font number ID
```
Comment (Jake-Carter) [inline Libraries/MiscDrivers/Display/tft_st7735.h:219]: Should be `@param font_id`, or rename the variable in the function declaration to `id`.

**Review by Jake-Carter — CHANGES_REQUESTED** [review]

File: Libraries/Boards/MAX32690/EvKit_V1/Source/board.c [hunk 2]
```diff
@@ -214,15 +213,10 @@ void TFT_SPI_Write(uint8_t *datain, uint32_t count, bool data)
                 tx_byte = tx_byte << 1;
             }
 
-            for (int k = 0; k < 2; k++) {}
```
[the remark this answers is not in the document]
Comment (EricB-ADI) [reply Libraries/Boards/MAX32690/EvKit_V1/Source/board.c:217]: I would be shocked if that for loop was not being optimized away.

**Comment by EricB-ADI** [comment]
Closing this as we no longer have access to the fork. Same changes in #1110

**Pull request closed without merging**